src/state/reducers/fetchingRecipesReducer.js

```diff
@@ -0,0 +1,24 @@
+import * as actionTypes from '../actions/actionTypes';
+
+const initialState = [{
```

Comment:
You initialState structure does not correspond to reducer model. You return an object from reducer, and here you have an array of objects.